Cargo.toml

```diff
@@ -1,31 +1,32 @@
 [package]
 name = "veridian"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Vivek Malneedi <vivekmalneedi@gmail.com>"]
 edition = "2018"
 
 [features]
 slang = ["veridian_slang"]
 
 [dependencies]
-sv-parser = "0.8.2"
+sv-parser = "0.13.0"
```

Comment:
Upgrading this is a little dangerous. I pushed my own attempt into the bump_deps branch, but the gist of it is that I think the version I was using hit a sweet spot where I could pass all my tests that involved parsing invalid code (by attempting to remove lines with syntax errors and rerunning the parse). After the upgrade that no longer seemed to be the case. There is a allow_incomplete flag now, which I think you set to true but that causes it to discard large parts of the syntax tree, which for the tests is effectively the entire tree. This might be worth the price regardless if the syntax tree it returns on "real" code is good enough for the language server to still be helpful.

The correct solution to this issue is to use a fault-tolerant IDE parser and I have new indexing code that uses treesitter to parse in the develop branch, but I haven't had time to actually rewrite the rest of veridian to accommodate it.

Reply:
The treesitter stuff is still quite a bit work away from working as part of this server, though I do hope to come back to it at some point. I'm just trying to give you some context for why fixing some of the tests could be non-trivial. I think for the purposes of this PR, the simplest option is to keep the allow_incomplete flag as true because it does solve certain issues and update to larger test snippets to prove that the resulting syntax tree is still useful (hopefully this is true, I haven't done much testing on it).